src/salesforce/profile.ts

```diff
@@ -63,16 +74,38 @@ export class SalesforceConnectorProfile extends ConnectorProfileBase {
     };
   }
 
+  /**
+   * This is a auxiliary method for obtaining a refreshTokeNGrandFlow. It's a temporary solution due to the typo in the properties
+   * as we don't want to abruptly fail customer solutions depending on the library.
+   * @param flow a SalesforceOAuthFlow object
+   * @returns a SalesforceOAuthRefreshTokenGrantFlow object or undefined if the flow is undefined.
+   * @throws an error if both refreshTokenGrant and refresTokenGrant are specified. This is a temporary solution due to the typo in the properties.
+   * @deprecated - this method will be removed in the future releases.
+   */
+  private getRefreshTokenGrantFlowProperty(flow?: SalesforceOAuthFlow): SalesforceOAuthRefreshTokenGrantFlow | undefined {
+
+    if (flow) {
+      if (flow.refresTokenGrant && flow.refreshTokenGrant) {
+        throw new Error('Only one of the properties refreshTokenGrant or refresTokenGrant should be specified');
+      }
+      return flow.refresTokenGrant ?? flow.refreshTokenGrant;
+    }
+    return undefined;
+  }
+
   protected buildConnectorProfileCredentials(properties: ConnectorProfileProps): CfnConnectorProfile.ConnectorProfileCredentialsProperty {
     const props = properties as SalesforceConnectorProfileProps;
 
     let salesforce: { [key: string]: any } = {};
 
+
     salesforce.accessToken = props.oAuth.accessToken;
-    salesforce.refreshToken = props.oAuth.flow?.refresTokenGrant.refreshToken ?? 'dummyRefreshToken';
```

Comment:
should dummy value be there? I'd suggest to use '' instead if want to keep default value.